README.md

```diff
@@ -1 +1,33 @@
-# pmm-submodules
\ No newline at end of file
+# PMM Submodules
+
+You can create feature build for PMM in this repo.
+
+## How to create feature build
+
+if you use the same branch name in all repos then you can run:
+```console
+make create <you branch name>
+```
+Branches with "you branch name" will be used for all repos or default branch if the branch with this name isn't found in repo.
+
+
+## Q&A
+### What if I need custom branch names for some repos?
+
+You can edit ci.yml by hand. For example:
+```yaml
+deps:
+  - name: global
+    branch: PMM-0000-fix-everything
+  - name: pmm-agent
+    branch: PMM-0000-fix-everything-and-even-more
+```
+Also, you need to add changes to git and push it:
+```
+git add ci.yml
+git commit -m 'use custom branches'
+git push
+```
+
+### What's global repo in ci.yml?
```

Comment:
having `global` on get deps staging adds more risks then benefits, I'm pretty sure that everybody will use it and it will be harder to figure out from submodules which repos were affected. So it brings the case when different team members might review only PRs requested from them without knowing about other PRs.

Reply:
We can send comment to PR with the list of branches for example. Or edit the first comment in PR?

Reply:
It's a hack, let's use git to trace changes instead of trying to hide them. If you want to make it easier for developers to rebuild PR in case of new repo added create new jenkins job or github action to rewrite ci.yml and commit.

Reply:
Changed and now global work only on the first run.